website/source/docs/commands/namespace/create.html.md.erb

```diff
@@ -0,0 +1,85 @@
+---
+layout: "docs"
+page_title: "Commands: Namespace Create"
+sidebar_current: "docs-commands-namespace-create"
+---
+
+<%= enterprise_alert :consul %>
+
+# Consul Namespace Create
+
+Command: `consul namespace create`
+
+This `namespace create` command creates a namespaces using the CLI parameters provided. 
+This was added in Consul Enterprise 1.7.2.
+
+## Usage
+
+Usage: `consul namespace create -name <namespace name> [options]`
+
+Request a namespace to be created. Construction of the namespace definition is handled by this command
+from the CLI arguments.
+
+#### API Options
+
+<%= partial "docs/commands/http_api_options_client" %>
+<%= partial "docs/commands/http_api_options_server" %>
+
+#### Command Options
+
+* `-default-policy-id=<value>` - ID of a policy from the default namespace to inject for all tokens
+   in this namespace. May be specified multiple times.
+
+* `-default-policy-name=<value>` - Name of a policy from the default namespace to inject for all
+   tokens in this namespace. May be specified multiple times.
+
+* `-default-role-id=<value>` - ID of a role from the default namespace to inject for all tokens in
+   this namespace. May be specified multiple times.
+
+* `-default-role-name=<value>` - Name of a role from the default namespace to inject for all tokens
+   in this namespace. May be specified multiple times.
+
+* `-description=<string>` - A description of the namespace.
+
+* `-format=<string>` - How to output the results. The choices are: pretty or json
+
+* `-meta=<value>` - Metadata to set on the namespace, formatted as key=value. This flag
+   may be specified multiple times to set multiple meta fields
+
+* `-name=<string>` - The namespace's name. This flag is required.
+
+* `-show-meta` - Indicates that namespace metadata such as the raft indices should
+   be shown for the namespace
+
+## Examples
+
+Create a new Namespace:
+
+```sh
+$ consul namespace create -name "team-1"
+Name: team-1
+```
+
+Showing Raft Metadata:
+
+```sh
+$ consul namespace crate -name team-1 -show-meta
+Name: team-1
+Create Index: 339
+Modify Index: 344
+```
+
+JSON Format:
+
+```sh
+$ consul namespace create -name foo -description "Example Namespace" -meta "team-id=574407f3-8b26-4c84-8e51-028bb8cbdd37" -format=json
+{
+   "Name": "foo",
```

Comment:
```suggestion
$ consul namespace create -name team2 -description "Example Namespace" -meta "team-id=574407f3-8b26-4c84-8e51-028bb8cbdd37" -format=json
{
   "Name": "team2",
```

foo is not descriptive, I think even team2 is better for this context. Entirely up to you to change it though.

Reply:
Works for me.